### PR TITLE
Replace get_storage_var_address function with the starknet-rs one (cont. closed PR #383)

### LINKED ISSUE #221: Replace get_storage_var_address
Replace `get_storage_var_address` function implementation with starknet_rs.
https://github.com/0xSpaceShard/starknet-devnet-rs/blob/bb22e929e6e6e666b9be4a5ce549933e08fd2ce7/crates/starknet/src/utils.rs#L18

### PR DESCRIPTION
**Pull request by devnet0x**
## Usage related changes

Mo user impact.

## Development related changes

Close #221 

## Checklist:

- [X] Checked the relevant parts of the [Development section of the docs](https://github.com/0xSpaceShard/starknet-devnet-rs/?tab=readme-ov-file#development)
- [X] Applied formatting - `./scripts/format.sh`
- [X] No linter errors - `./scripts/clippy_check.sh`
- [X] No unused dependencies - `./scripts/check_unused_deps.sh`
- [X] Performed code self-review
- [X] Rebased to the last commit of the target branch (or merged it into my branch)
- [N/A] Documented the changes
- [X] Checked if the [TODO section of the docs](https://github.com/0xSpaceShard/starknet-devnet-rs/?tab=readme-ov-file#todo-to-reach-feature-parity-with-the-pythonic-devnet) can be edited
- [X] Linked the issues which this PR resolves
- [X] Updated the tests if needed; all passing ([execution info](https://github.com/0xSpaceShard/starknet-devnet-rs/?tab=readme-ov-file#test-execution))

### CODE REVIEW

File: crates/starknet-devnet-core/src/starknet/predeployed.rs [hunk 1]
```diff
@@ -51,10 +51,12 @@ pub(crate) fn initialize_erc20_at_address(
         // necessary to set - otherwise minting txs cannot be executed
         ("Ownable_owner", Felt::from_prefixed_hex_str(CHARGEABLE_ACCOUNT_ADDRESS)?),
     ] {
-        let storage_var_address = get_storage_var_address(storage_var_name, &[])?.try_into()?;
+        let storage_var_address =
+            PatriciaKey::try_from(get_storage_var_address(storage_var_name, &[]).unwrap()).unwrap();
```
Comment (FabijanC) [inline crates/starknet-devnet-core/src/starknet/predeployed.rs:55]: More unwrapping.

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: crates/starknet-devnet-core/src/starknet/predeployed.rs (hunk 1 above)
Comment (devnet0x) [reply crates/starknet-devnet-core/src/starknet/predeployed.rs:55]: please could you give me some advice @FabijanC ?...i don't know what else to do to deal with these unwraps. Thanks in advance.

Comment (ivpavici) [reply crates/starknet-devnet-core/src/starknet/predeployed.rs:55]: please take your time, learn a bit, research and try! For example, one pointer:
image

Comment (devnet0x) [reply crates/starknet-devnet-core/src/starknet/predeployed.rs:55]: great thanks

Comment (devnet0x) [reply crates/starknet-devnet-core/src/starknet/predeployed.rs:55]: Hi @ivpavici , no i don't know how to handle this issue.

**Comment by FabijanC** [comment]
Closed in favor of #440

**Pull request closed without merging**